hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SequenceIdGenerator.java

```diff
@@ -253,9 +254,28 @@ public Long getLastId(String sequenceIdName) {
     }
 
     @Override
-    public void reinitialize(Table<String, Long> seqIdTable) {
+    public void reinitialize(Table<String, Long> seqIdTable)
+        throws IOException {
       this.sequenceIdTable = seqIdTable;
       this.sequenceIdToLastIdMap.clear();
+      initialize();
+    }
+
+    private void initialize() throws IOException {
+      TableIterator<String,
+          ? extends Table.KeyValue<String, Long>>
+          iterator = sequenceIdTable.iterator();
+
+      while (iterator.hasNext()) {
+        Table.KeyValue<String, Long> kv = iterator.next();
+        final String sequenceIdName = kv.getKey();
+        final Long lastId = kv.getValue();
+        org.apache.ratis.util.Preconditions.assertNotNull(sequenceIdName,
```

Comment:
Can u fix the import and use just Preconditions check?

Reply:
sure, will do this